hutch_python/obj_config.py

```diff
@@ -0,0 +1,186 @@
+import logging
+from pathlib import Path
+from typing import Callable, Tuple, Union
+
+import yaml
+from ophyd import Device, Kind
+from pcdsdevices.interface import BaseInterface
+
+from . import utils
+
+logger = logging.getLogger(__name__)
+
+
+def update_objs(
```

Comment:
Does this function need to show warnings when we didn't find any matches?

Imagine I misspell an object or a class name and then my tab settings don't apply. Is this something we need to worry about happening?

Reply:
Definitely.  A good thing to add